contract/models/contract.py

```diff
@@ -574,6 +574,9 @@ def recurring_create_invoice(self):
         This method triggers the creation of the next invoices of the contracts
         even if their next invoicing date is in the future.
         """
+        _logger.warning(
+            "recurring_create_invoice is no longer necessary. It should be removed on v17"
```

Comment:
I've the feeling that we should remove it if it's not used anymore ?

Other wise I would expect kind of clear message such as:
```suggestion
            "recurring_create_invoice is deprecated in favor of _recurring_create_invoice instead"
```

Reply:
Yes, it is much cleaner :smile: